### PR TITLE
Fix/permission

### LINKED ISSUE #6769: ExplorerVisible @ permission
### What happened?

explorerVisible function is missing

2 files still use exporerVisible
image

also, permision.config has a wrong spelling will be renaming it to
permission.config

This issue was caused by https://github.com/kodadot/nft-gallery/pull/6748/
### Please reproduce in steps

Open '@/utils/config/permision.config'
https://github.com/kodadot/nft-gallery/blob/main/utils/config/permision.config.ts
and find explorerVisible

### Expected Behavior

explorerVisible should exist

### What browsers are you seeing the problem on?

_No response_

### At which address did you encounter bug?

_No response_

### Are you logged in?

None

### Which wallet you are using?

_No response_

### At which chain did you encounter bug?

_No response_

### Screenshots

image

### Relevant log output

_No response_

### Payment link for reward

https://canary.kodadot.xyz/dot/transfer?target=16UcV9V6nVvPYdHz98ymUKmNLkzjCEU5sbKJMi7hxYyTHjzR&usdamount=100&donation=true

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by stephenjason89**
## PR Type

- [x] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #6769 related to https://github.com/kodadot/nft-gallery/pull/6748?notification_referrer_id=NT_kwDOADojgbI3NDEzNzMwODE5OjM4MTAxNzc#issuecomment-1683219606
- [ ] Requires deployment <snek/rubick/worker>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality

#### Optional

- [ ] I've tested it at </ksm/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer?target=16UcV9V6nVvPYdHz98ymUKmNLkzjCEU5sbKJMi7hxYyTHjzR&usdamount=100&donation=true)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at 769abca</samp>

Refactored the code for opening the shopping cart modal from the navbar. Moved the logic and configuration from `ShoppingCartModalConfig.ts` and `ShoppingCartButton.vue` to `Navbar.vue` to improve code organization and performance.

### <samp>🤖 Generated by Copilot at 769abca</samp>

> _`ShoppingCartButton`_
> _Simpler, no more `emit` -_
> _Autumn leaves falling_

### CODE REVIEW

File: pages/sales.vue [hunk 1]
```diff
@@ -8,7 +8,7 @@
   </section>
 </template>
 <script lang="ts">
-import { salesVisible } from '@/utils/config/permision.config'
+import { salesVisible } from '~/utils/config/permission.config'
```
Comment (preschian) [inline pages/sales.vue:11]: just keep it using `@` instead of `~`

File: utils/config/permission.config.ts
Comment (preschian) [inline utils/config/permission.config.ts:1]: need more files to rename, you can check the error log on the build https://app.netlify.com/sites/koda-canary/deploys/64ded3bb3a5cb70009759461

start with `ERROR in`

File: pages/sales.vue (hunk 1 above)
Comment (stephenjason89) [reply pages/sales.vue:11]: @preschian oops sorry my editor automatically renamed the files didn't notice it was changed to ~
Thank you for pointing that out

File: utils/config/permission.config.ts
Comment (stephenjason89) [reply utils/config/permission.config.ts:1]: @preschian Done. It was unstaged on my git haha silly me . Sorry now everything should be good to go.

**Review by preschian — APPROVED** [review]

**Review by roiLeo — APPROVED** [review]

**Review by vikiival — CHANGES_REQUESTED** [review]
https://github.com/kodadot/nft-gallery/pull/6748#issuecomment-1681977022

**Comment by stephenjason89** [comment]
> https://github.com/kodadot/nft-gallery/pull/6748#issuecomment-1681977022

I completely agree on this one. Will push changes in a while.

**Comment by stephenjason89** [comment]
@vikiival I implemented the type something along the lines that you suggested. Had to correct it  a bit to make it work perfectly.

**Review by vikiival — APPROVED** [review]

File: utils/config/permission.config.ts [hunk 2]
```diff
@@ -1,67 +1,50 @@
...
-const hasInsight: Config<boolean> = {
-  rmrk: true,
-  ksm: true,
-  bsx: true,
+const hasInsight: PartialConfig = {
   ahk: false,
   movr: false,
   glmr: false,
-  snek: true,
-  dot: true,
```
Comment (vikiival) [inline utils/config/permission.config.ts:23]: DOT is also false

Comment (stephenjason89) [reply utils/config/permission.config.ts:23]: done

**Comment by yangwao** [comment]
@stephenjason89 let's resolve conflicts

**Review by vikiival — APPROVED** [review]

**Comment by stephenjason89** [comment]
@yangwao Thanks, done 😄

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
Cc @kodadot/qa-guild

**Comment by vikiival** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 4.48 USD/DOT ~ 6.696 $DOT
🧗 [16UcV9V6nVvPYdHz98ymUKmNLkzjCEU5sbKJMi7hxYyTHjzR](https://polkadot.subscan.io/account/16UcV9V6nVvPYdHz98ymUKmNLkzjCEU5sbKJMi7hxYyTHjzR)
🔗 [0xb9a6f50546a0202b30803b7dd010c3c523c0699512cd7d02c8bff7145f498f16](https://polkadot.subscan.io/extrinsic/0xb9a6f50546a0202b30803b7dd010c3c523c0699512cd7d02c8bff7145f498f16)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**